observability/monitoring/common-monitoring-configuration-scenarios.adoc

```diff
@@ -0,0 +1,53 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="common-monitoring-configuration-scenarios"]
+= Common monitoring configuration scenarios
+include::_attributes/common-attributes.adoc[]
+:context: common-monitoring-configuration-scenarios
+
+toc::[]
+
+The default platform monitoring stack is available for use after installing {product-title}.
+The in-cluster stack includes the core platform Prometheus instance that collects metrics from your cluster and the core Alertmanager instance that routes alerts.
+
+In addition to core platform monitoring, you can also optionally xref:../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[enable monitoring for user-defined projects] for user workload monitoring.
+After doing so, users can then monitor their own services and workloads without the need for an additional monitoring solution.
+
+Depending on which users will use the monitoring stack and for what purposes, as a cluster administrator, you can configure monitoring components to suit the needs of different users in various scenarios.
+
+[id="core-platform-monitoring-first-steps_{context}"]
+== Core platform monitoring: First steps
+
+After {product-title} is installed, cluster administrators typically perform certain basic activities to configure core platform monitoring to suit their needs.
+These activities include configuring options for Prometheus and Alertmanager, setting up short-term and long-term storage, and adding service monitors.
+When you have completed these configuration activities, users can query and view collected metrics and receive alerts.
```

Comment:
_Nit_ Metrics viewing and querying works out of the box. Only an alert receiver must be configured in order to receive the alert notifications.

Reply:
Reworded to make this clear. Also added a clarifying note to the beginning